index.html

```diff
@@ -159,141 +156,401 @@ <h2>
-            gesture</a> must be received by the document before
-            {{requestPointerLock()}} will succeed.
-          </p>
-          <aside class="note" data-link-for="Document">
-            <p>
-              This ensures a user can leave a document that constantly attempts
-              to lock the pointer. The document will be blocked from locking
-              upon initial navigation or re-acquiring lock unless the user
-              re-engages directly with the document.
-            </p>
-            <p>
-              Conversely, if pointer lock is exited via {{exitPointerLock()}}
-              no <a>engagement gesture</a> is required to reenter pointer lock.
-              This enables applications that frequently move between
-              interaction modes, and ones that may do so based on a timer or
-              remote network activity.
-            </p>
-            <p data-link-for="Element">
-              Pointer lock is commonly combined with fullscreen [[FULLSCREEN]],
-              and if an <a>engagement gesture</a> is used to enter fullscreen
-              it is sufficient for a subsequent {{requestPointerLock()}}.
-            </p>
-          </aside>
-          <p>
-            If any element (including this one), whose <a>shadow-including
-            root</a> is same as this element's <a>shadow-including root</a>, is
-            already locked (or pending lock) the pointer lock <a>target</a>
-            must be updated to this element and a <a>pointerlockchange</a>
-            event sent.
-          </p>
-          <p>
-            If any element, whose <a>shadow-including root</a> is a different
-            document, is already locked the request must fail and a
-            <a>pointerlockerror</a> event be sent.
-          </p>
-          <p>
-            Once in the locked state the <a>user agent</a> must fire all
-            relevant user generated {{MouseEvent}} events (for example:
-            `mousemove`, `mousedown`, `mouseup`, `click`, and `wheel`)
-            [[ui-events]] to the <a>target</a> of pointer lock, and not fire
-            mouse events to other elements. Events that require the concept of
-            a mouse cursor must not be dispatched (for example: `mouseover`,
-            `mouseout`, `drag`, and `drop`).
-          </p>
-          <p>
-            In the locked state the system mouse cursor must be hidden.
-            Movement and button presses of the mouse must not cause the window
-            to lose focus.
-          </p>
-          <p>
-            Synthetic mouse events created by application script act the same
-            regardless of lock state.
+            <br>
+            When <dfn>requestPointerLock</dfn> invoked, perform the following
+            steps:
           </p>
+          <ol>
+            <li>Let <var>promise</var> be [=a new promise=].
+            </li>
+            <li>When a window is in focus, if the [=this=]'s [=shadow-including
```

Comment:
```suggestion
            <li>When a [=window=] is in focus, if the [=this=]'s [=shadow-including
```

Reply:
Can we link to focus somehow/somewhere I wonder? I wonder if HTML defines something. 

It defines "system focus"... maybe that's the right one?

Reply:
I think [=Window/focus=] is what here is referring to.